# msg/MotorCommandArray.msg
robot_driver_pkg/MotorCommand[] commands
string[] joint_names
